### ISSUE TITLE
Is there a way to disable the Live Render in 3.0

### ISSUE THREAD

**Issue by traber333** [swagger-api/swagger-editor#1422]
When working on large files the Live Render makes it nearly impossible to type.  In v2.x of the editor there was a preference for turning off Live Rendering, but I can't seem to find anything like this in the current 3.x version.  Is there a plan to put this back or some way for me to disable the Live Rendering and Validation?

**Comment by shockey** [comment]
@traber333, good idea 😄 keep in mind that PRs for functionality are welcome, if you have the time!

Also, if you can provide the spec that's giving you problems, that would help us debug what's making the Editor slow for you.

**Comment by traber333** [comment]
@shockey Sorry for the delay in getting back to you.  My spec file is over 6000 lines long so I think that may be the reason.  It is attached to this reply.
[swagger.yaml.zip](https://github.com/swagger-api/swagger-editor/files/1214653/swagger.yaml.zip)

**Comment by shockey** [comment]
@traber333, FYI we pushed out a performance release last week that has made the Editor much faster!

**Comment by DmitryMelnyk** [comment]
My spec file is over 8k lines long and it will grow up to 10k soon. So disabling the Live Render feature is a must

**Comment by smagret** [comment]
Hi, any news on this? I'm having the same issue with a 10k lines file. With the live render it makes it almost impossible to edit the file :(

**Comment by MichakrawSB** [comment]
We’ve released Swagger Editor v5! 🎉 Check out the details here: [Inside the New Swagger Editor](https://swagger.io/blog/inside-the-new-swagger-editor/).
We’re closing old issues related to previous versions. If you think any of them are still relevant, please open a new issue – this helps us prioritize what matters most. Thanks for your activity! 🚀
